internal/migration/migration_test.go

```
@@ -0,0 +1,346 @@
+
+			if originalMR.Spec.MySQL != nil && convertedMR.Spec.MySQL != nil {
+				Expect(convertedMR.Spec.MySQL.Host).To(Equal(originalMR.Spec.MySQL.Host))
+				Expect(convertedMR.Spec.MySQL.Database).To(Equal(originalMR.Spec.MySQL.Database))
+			}
+		})
+	})
+})
+
+// Helper function to generate random strings for test names
+func randStringRunes(n int) string {
+	rand.Seed(time.Now().UnixNano())
```

golangci-lint (1.64.8)

SA1019: rand.Seed has been deprecated since Go 1.20 and an alternative has been available since Go 1.0: As of Go 1.20 there is no reason to call Seed with a random value. Programs that call Seed with a known value to get a specific sequence of results should use New(NewSource(seed)) to obtain a local random generator.

(staticcheck)